src/languages/maxima.js

```diff
@@ -0,0 +1,415 @@
+      + ' ratsimpexpons ratvarswitch ratweights ratweyl ratwtlvl real realonly redraw'
+      + ' refcheck resolution restart resultant ric riem rmxchar %rnum_list rombergabs'
+      + ' rombergit rombergmin rombergtol rootsconmode rootsepsilon run_viewer same_xy'
+      + ' same_xyz savedef savefactors scalar scalarmatrixp scale scale_lp setcheck'
+      + ' setcheckbreak setval show_edge_color show_edges show_edge_type show_edge_width'
+      + ' show_id show_label showtime show_vertex_color show_vertex_size show_vertex_type'
+      + ' show_vertices show_weight simp simplified_output simplify_products simpproduct'
+      + ' simpsum sinnpiflag solvedecomposes solveexplicit solvefactors solvenullwarn'
+      + ' solveradcan solvetrigwarn space sparse sphere spring_embedding_depth sqrtdispflag'
+      + ' stardisp startphi starttheta stats_numer stringdisp structures style sublis_apply_lambda'
+      + ' subnumsimp sumexpand sumsplitfact surface surface_hide svg_file symmetric'
+      + ' tab taylordepth taylor_logexpand taylor_order_coefficients taylor_truncate_polynomials'
+      + ' tensorkill terminal testsuite_files thetaresolution timer_devalue title tlimswitch'
+      + ' tr track transcompile transform transform_xy translate_fast_arrays transparent'
+      + ' transrun tr_array_as_ref tr_bound_function_applyp tr_file_tty_messagesp tr_float_can_branch_complex'
+      + ' tr_function_call_default trigexpandplus trigexpandtimes triginverses trigsign'
+      + ' trivial_solutions tr_numer tr_optimize_max_loop tr_semicompile tr_state_vars'
+      + ' tr_warn_bad_function_calls tr_warn_fexpr tr_warn_meval tr_warn_mode'
+      + ' tr_warn_undeclared tr_warn_undefined_variable tstep ttyoff tube_extremes'
+      + ' ufg ug %unitexpand unit_vectors uric uriem use_fast_arrays user_preamble'
+      + ' usersetunits values vect_cross verbose vertex_color vertex_coloring vertex_partition'
+      + ' vertex_size vertex_type view warnings weyl width windowname windowtitle wired_surface'
+      + ' wireframe xaxis xaxis_color xaxis_secondary xaxis_type xaxis_width xlabel'
+      + ' xlabel_secondary xlength xrange xrange_secondary xtics xtics_axis xtics_rotate'
+      + ' xtics_rotate_secondary xtics_secondary xtics_secondary_axis xu_grid x_voxel'
+      + ' xy_file xyplane xy_scale yaxis yaxis_color yaxis_secondary yaxis_type yaxis_width'
+      + ' ylabel ylabel_secondary ylength yrange yrange_secondary ytics ytics_axis'
+      + ' ytics_rotate ytics_rotate_secondary ytics_secondary ytics_secondary_axis'
+      + ' yv_grid y_voxel yx_ratio zaxis zaxis_color zaxis_type zaxis_width zeroa zerob'
+      + ' zerobern zeta%pi zlabel zlabel_rotate zlength zmin zn_primroot_limit zn_primroot_pretest';
+
+  return {
+    lexemes: '[A-Za-z_%][0-9A-Za-z_%]*',
+    keywords: {
+      keyword: KEYWORDS,
+      literal: LITERALS,
+      built_in: BUILTIN_FUNCTIONS,
+      symbol: BUILTIN_VARIABLES
```

Comment:
I'm afraid that'd go against some of the design decisions in highlight.js.

One of the goals is for highlighting to be consistent across languages, so we have a limited set of stylable features (class names) that style authors rely on when producing styles. Re-using a class name for another purpose may cause undesired effects and adding a new class name means updating all of the existing styles and complicating creation of new ones. I would agree though, that in this particular case styling built-in vars as "symbol" probably won't cause any artistic shock anywhere :-) But this does explain why we don't simply treat each language individually.

However! There's another design choice of keeping highlighting minimalistic (and more useful). This is subjective, of course, and in this case I simply follow a precedent set in 9739df60a: upper-case gives enough distinction to those vars.